mobile/lib/screens/settings/settings_page.dart

```diff
@@ -1,3 +1,5 @@
+import 'dart:async';
+
```

Comment:
What is the purpose of this import? @katendejericho5

Reply:
Future<void> _deleteAccount(BuildContext context) async {
it is being used here